test/io/input/pipeprocesstester.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+sum_to () (
+    set -- $(seq $1)
+    IFS=+
```

Comment:
I just wanted some kind of function that takes some time to run. My only critera was that it does not contain `rm -rf /`. This did the trick. :)